Patch to fix two issues:
- upstream does: CC = g++
In order to keep the Makefile the same we set CC = $(CXX)
- install target  
Add DESTDIR so that the destroot phase works properly

--- Makefile.orig	2020-03-04 21:45:27.000000000 -0500
+++ Makefile	2020-03-04 21:50:00.000000000 -0500
@@ -1,6 +1,4 @@
-CC = g++ -Wall -ggdb
-CC = g++ -pg
-CC = g++
+CC = $(CXX)
 
 # default with OpenMP
 # with OpenMP
@@ -99,10 +97,10 @@
 cdhit-454.o: cdhit-454.c++ cdhit-common.h
 	$(CC) $(CCFLAGS) cdhit-454.c++ -c
 
-PREFIX ?= /usr/local/bin
+PREFIX ?= @@PREFIX@@
 
 install:
 	for prog in $(PROGS); do \
-		install -m 0755 $$prog $(PREFIX); \
+		install -m 0755 $$prog $(DESTDIR)$(PREFIX)/bin; \
 	done
-	install -m 0755 *.pl $(PREFIX);
+	install -m 0755 *.pl $(DESTDIR)$(PREFIX)/bin;
